uwblueprint/childrens-aid-society
# create custom input and change all inputs to custom

## Notion ticket link

[Create Custom Input Component](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* created a custom input component under common components
* added forwardRef so that we can pass in the ref prop needed for attaching a new file in CourtDetailsForm.tsx


## Steps to test
1. checkout branch
2. check homepage for input with an icon and intake page for input without an icon

<img width="835" alt="Screen Shot 2022-08-17 at 9 24 37 PM" src="https://user-images.githubusercontent.com/77185514/185271680-a75f1cf7-085b-47f5-b58c-a3a8661d76e0.png">



## What should reviewers focus on?
* if the component is placed in the right spot
* if the component looks good -> i had to disable eslint to pass in the props using the spread operator (since that's the easiest way to pass props into the Input component without making anything mandatory/missing any props)


## Checklist
